src/pages/FrontPage.js

```diff
@@ -37,16 +37,18 @@ export const  FrontPage = React.createClass({
         </Row>
 
         {/* Buttons */}
-        <Row>
-          <div className={styles.center}>
-            <ButtonItem color='green' onClick={() => this.displayExercise}>
-              Kom i gang!
-            </ButtonItem>
-            <ButtonItem color='blue' onClick={() => this.props.changeMode()}>
-              Lærer/Veileder
-            </ButtonItem>
-          </div>
-        </Row>
+        {this.props.isStudentMode ?
+          <Row>
+            <div className={styles.center}>
+              <ButtonItem color='green' onClick={() => this.displayExercise}>
+                Kom i gang!
+              </ButtonItem>
+              <ButtonItem color='blue' onClick={() => this.props.changeMode()}>
```

Comment:
Vil det bli mer tydelig hva denne knappen gjør hvis `this.props.setModeTeacher()` blir brukt i stedet for `changeMode()`? (Siden denne knappen alltid skal sette mode til teacherMode)